packages/editor/src/components/inserter/menu.js

```diff
@@ -48,14 +49,29 @@ const MAX_SUGGESTED_ITEMS = 9;
  * @return {Array}             Filtered item list.
  */
 export const searchItems = ( items, searchTerm ) => {
-	const normalizedSearchTerm = searchTerm.toLowerCase().trim();
-	const matchSearch = ( string ) => string.toLowerCase().indexOf( normalizedSearchTerm ) !== -1;
+	const normalizedSearchTerm = normalizeTerm( searchTerm );
+	const matchSearch = ( string ) => normalizeTerm( string ).indexOf( normalizedSearchTerm ) !== -1;
 
 	return items.filter( ( item ) =>
 		matchSearch( item.title ) || some( item.keywords, matchSearch )
 	);
 };
 
+/**
+ * Converts the search term into a normalized term, removing diacritics
+ * and whitespace.
+ *
+ * @param {string} term The search term to normalize
+ * @return {string} The normalized search term
+ */
+export const normalizeTerm = ( term ) => {
+	term = deburr( term );
+	term = term.toLowerCase();
+	term = term.trim();
+
+	return term;
```

Comment:
Couldn't this just be simplified to `return deburr( term ).toLowerCase().trim()` and still be as readable?

Reply:
Yup, it could be. I've left it separate to allow for additional normalizations. I suspect there will be a few more like #8466.

Reply:
I'm personally fine with this. If we want to get fancy-functional, we could also use [Lodash's `_.flow`](https://lodash.com/docs/4.17.10#flow) to compose them:

```js
export const normalizeTerm = flow( [
	deburr,
	lowerCase,
	trim,
] );
```

- https://lodash.com/docs/4.17.10#lowerCase
- https://lodash.com/docs/4.17.10#trim